composer.json

```diff
@@ -23,6 +23,7 @@
         "psr/container": "^1.0",
         "psr/log": "~1.0",
         "psr/simple-cache": "^1.0",
+        "fig/link-util": "^1.0",
```

Comment:
How easy it is to define a virtual package (basically just list it in `provide` - that's why it's called virtual):
https://devedge.wordpress.com/2014/09/27/composer-and-virtual-packages/

And some counter points:
https://php-and-symfony.matthiasnoback.nl/2014/10/composer-provide-and-dependency-inversion/